### PR TITLE
[Update] Pebblo UI documentation added.

### PR DESCRIPTION
**Pull request by KumarNitin19**
- Pebblo UI documentation added.

### CODE REVIEW

**Review by rutujaac — DISMISSED** [review]
LGTM

**Review by shreyas-damle — CHANGES_REQUESTED** [review]
Please rebase this with main and add UI as a sub section within Pebblo section.

File: docs/gh_pages/docs/installation.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ pip install pebblo
 pebblo
 ```
 
-Pebblo server now listens to `localhost:8000` to accept Gen-AI application data snippets for inspection and reporting.
+Pebblo server now listens to `localhost:8000` to accept Gen-AI application data snippets for inspection and reporting. Pebblo UI interface would be available on `http://localhost:8000/pebblo `
```
[the remark this answers is not in the document]
Comment (KumarNitin19) [reply docs/gh_pages/docs/installation.md]: @shreyas-damle Comment resolved

File: docs/gh_pages/docs/config.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ This configuration file specifies settings for various components of the Pebblo.
 - `port`: Specifies the port number on which the Pebblo server listens for incoming connections.
 - `host`: Specifies the host address on which the Pebblo server to run.
 
-Note: By default `Pebblo Server` runs at `localhost:8000`. When we change values of `port` and/or `host` , the `Pebblo Safe DataLoader` env variable `PEBBLO_CLASSIFIER_URL` needs to set to the correct URL.
+Notes:
+1. By default `Pebblo Server` runs at `localhost:8000`. When we change values of `port` and/or `host` , the `Pebblo Safe DataLoader` env variable `PEBBLO_CLASSIFIER_URL` needs to set to the correct URL.
+2. By default `Pebblo UI` runs at `localhost:8000/pebblo`. When we change values of `port` and/or `host`, the Pebblo UI would be running on the respective `url/pebblo`.
```
[the remark this answers is not in the document]
Comment (KumarNitin19) [reply docs/gh_pages/docs/config.md]: @shreyas-damle Comment resolved

**Review by shreyas-damle — APPROVED** [review]

**Review by rutujaac — APPROVED** [review]

**Pull request merged**